1-exercises/A-reading-list/script.js

```diff
@@ -1,27 +1,52 @@
 function readingList(books) {
-  // Write your code here...
+  // Write your code here..
+  const getParentElement = document.querySelector("#content");
+  
+  const ulElement = document.createElement("ul");
+
+  for (let book of books) {
+    const liElement = document.createElement("li");
+
+    const pElement = document.createElement("p");
+    pElement.innerText = `${book.title} By ${book.author}`;
+    liElement.appendChild(pElement);
+
+    if (book.alreadyRead === true) {
+      liElement.style.backgroundColor = "green";
+    } else {
+      liElement.style.backgroundColor = "red";
+    }
+
+    ulElement.appendChild(liElement);
+
+    const imgElement = document.createElement("img");
+    imgElement.src = book.bookCoverImage;
+    liElement.appendChild(imgElement);
+  }
+  getParentElement.appendChild(ulElement);
+  ulElement.style.width = 90 + "vw"
```

Comment:
Nice touch :) Page works correctly, very good! The code quality is good except for the places where you've done the things I mentioned above